resources/NBTest/NBTest_903_FlashloanTokens.py

```diff
@@ -83,7 +83,7 @@ def run_command(mode):
     # Wait for the expected log line to appear
     expected_log_line = "limiting flashloan_tokens to ["
     found = False
-    result = subprocess.run(cmd, text=True, capture_output=True, check=True)
+    result = subprocess.run(cmd, text=True, capture_output=True, check=True, timeout=120)
```

Comment:
making sure we're not increasing timeout to workaround an actual performance issue that needs addressing